phalcon/Db/Adapter/AbstractAdapter.zep

```diff
@@ -4,8 +4,8 @@
  *
  * (c) Phalcon Team <team@phalcon.com>
```

Comment:
Seems that current licence is not the latest that we are using.

Reply:
Not only this :)